examples/ioctl.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+# Example code for handling IOCTLs in FUSE client
+
+import ctypes
+import struct
+from errno import ENOTTY
+from ioctl_opt import IOWR
+
+def ioctl(self, path, cmd, arg, fh, flags, data):
```

Comment:
Can you define the whole operations class? As it stands this example "does not work"